Assets/Scripts/Augment/BulletModifiers/KnockbackOnShotModifier.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class KnockbackOnShotModifier: MonoBehaviour, ProjectileModifier
+{
+    [SerializeField]
+    private float pushPower;
+
+    [SerializeField]
+    private float radius;
+
+    [SerializeField]
+    private KnockbackEffect knockBackScript;
```

Comment:
Please don't use "script" in names for component references. Everything is a script (or class) here. (don't get me started on the fact that there are C# scripts (csx) now that are a very different thing)